spec/integration/network/http/api/indirected_routes_spec.rb

```diff
@@ -2,6 +2,8 @@
 require 'spec_helper'
 require 'puppet/network/http'
 require 'puppet/network/http/api/indirected_routes'
+require 'rack/mock' if Puppet.features.rack?
```

Comment:
As discussed in meatspace, I think is fine and not really worth changing at this point.